src/utils/index.js

```diff
@@ -1,2 +1,4 @@
-export { default as camelizeKeys } from './camelizeKeys';
-export { default as toFormData } from './toFormData';
+import { camelizeKeys } from './camelizeKeys';
+import { toFormData } from './toFormData';
+
```

Comment:
Was change required in this file?

Reply:
Sir, I thought it would be more consistent to the rest of the codebase. Should I revert the change?

Reply:
Yes, I think you should revert the changes of the file. The current approach of exporting is better,